engine/src/main/java/org/terasology/engine/subsystem/lwjgl/BaseLwjglSubsystem.java

```diff
@@ -15,6 +15,8 @@
  */
 package org.terasology.engine.subsystem.lwjgl;
 
+import com.badlogic.gdx.physics.bullet.Bullet;
+import com.badlogic.gdx.physics.bullet.BulletBase;
```

Comment:
Just looking on GitHub, but it doesn't seem like this import gets used? Also is a LWJGL subsystem the best place for physics? It might be, but it the physics isn't strictly related to LWJGL.